### PR TITLE
Replace occurrences of OpenRCT with OpenRCT2

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

**Comment by Gymnasiast** [comment]
The parameter change (`openrct-data-path`) should be put into the changelog.

**Comment by IntelOrca** [comment]
The man page needs updating too.

**Comment by Gymnasiast** [comment]
@IntelOrca Thanks! Changed that one too. I also did a search for `openrct[^2]` and fixed some other occurrences as well.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - Change: [#11209] Warn when user is running OpenRCT2 through Wine.
 - Change: [#11358] Switch copy and paste button positions in tile inspector.
 - Change: [#11449] Remove complete circuit requirement from Air Powered Vertical Coaster (for RCT1 parity).
+- Change: [#11898] The `openrct-data-path` parameter has been renamed to `openrct2-data-path`.
```
Comment (AaronVanGeffen) [inline distribution/changelog.txt:16]: Perhaps the following is clearer:
```suggestion
- Change: [#11898] The `openrct-data-path` command-line argument has been renamed to `openrct2-data-path`.
```

**Pull request merged**